### PR TITLE
Added password prompt and ability to upload to SSM. Password is also …

### PR DESCRIPTION
**Pull request by ggotti**
- Added support for using Amazon SSM for storing database passwords.
- Removed hard-coding of password within CloudFormation file

### CODE REVIEW

File: generators/more-aws/index.js [hunk 1]
```diff
@@ -304,11 +305,27 @@ module.exports = class extends BaseGenerator {
                         done();
                     });
             },
+            setSSMDatabasePassword() {
+                if (this.abort || this.awsFacts.database_Password_InSSM) return null;
+                const done = this.async;
+                const passwordProperty = `${this.aws.cloudFormationName}.spring.datasource.password`;
+                return awsClient.setSSMParameter(
+                    passwordProperty,
+                    this.awsFacts.database_Password, `Database master password for ${this.aws.cloudFormationName}`
```
Comment (carvallegro) [inline generators/more-aws/index.js]: Either do a new line for every parameter or for none of them

File: generators/more-aws/index.js [hunk 2]
```diff
@@ -52,6 +52,8 @@ module.exports = class extends BaseGenerator {
                         password: null
                     },
                     s3BucketName: awsClient.sanitizeBucketName(`${this.baseName}_${new Date().getTime()}`),
+                    taskCount: 1,
+                    database_Username: `${this.baseName}_db_user`,
```
Comment (carvallegro) [inline generators/more-aws/index.js]: `database_Username` is not camelCase. It's more like camel cooked façon kebab_case. Please don't kill the camel

Comment (ggotti) [reply generators/more-aws/index.js]: 🌮

File: generators/more-aws/index.js (hunk 1 above)
Comment (ggotti) [reply generators/more-aws/index.js]: :Taco

**Pull request merged**